### DISCUSSION TITLE
Hi, great work!!!

### DISCUSSION THREAD

**Discussion by linuxdigital** [julian-poidevin/MBPMid2010_GPUFix#158] [General]
How to run the app?

**Comment by julian-poidevin** [comment]
Hello,

1. First disable SIP on your machine by following this tutorial
2. Download the application using this link :  https://github.com/julian-poidevin/MBPMid2010_GPUFix/releases/download/v0.6.1/MBPMid2010-GPU-Fix-0.6.1.zip
3. Extract the ZIP
4. Clock on the application icon
5. Click on "Patch MBP"
6. If prompted a warning message, press "Yes".
7. If prompted a password request, type your user password, the one you use the access your session.
8. Wait until the process finished
9. Reboot your computer and you should be done.
